.circleci/config.yml

```diff
@@ -21,7 +21,8 @@ references:
     run:
       name: Install Code Climate test-reporter
       command: |
-        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > tmp/cc-test-reporter
+        # curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > tmp/cc-test-reporter
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-0.7.0-linux-amd64 > tmp/cc-test-reporter
```

Comment:
Need to remember to revert this out at some point

Reply:
@jrmhaig I think the issue with test-reporter has been resolved and this can be removed